packages/mgt-components/src/components/mgt-person-card/mgt-person-card.scss

```diff
@@ -396,6 +392,10 @@ $person-card-base-icons-left-spacing: var(--person-card-base-icons-left-spacing,
   --person-avatar-size-3-lines: 75px;
 }
 
+:host fluent-tabs.horizontal::part(activeIndicator) {
```

Comment:
I don't understand why this is split into a separate selector here, could make use of nesting still if you need this exact selector using the `&` operator in scss to combine the nested selector fragments together

Reply:
Definitely shouldn't be split, changing this.